mcs/class/corlib/System.Threading/Thread.cs

```diff
@@ -321,6 +321,9 @@ static void SerializePrincipal (Thread th, IPrincipal value)
 			}
 		}
 
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        public static extern long GetAllocatedBytes();
```

Comment:
There is already `GC.GetAllocatedBytesForCurrentThread()` for it. But with empty icall (https://github.com/mono/mono/issues/8397)

Reply:
There is already existing API for this which is wired up to https://github.com/mono/mono/blob/e2d6361771b489706ab0264b30e8040f20060bd8/mono/metadata/icall.c#L9177, you need to hook your code there instead